build.gradle

```diff
@@ -4,10 +4,10 @@ plugins {
   id 'pmd'
   id 'jacoco'
   id 'io.spring.dependency-management' version '1.1.7'
-  id 'org.springframework.boot' version '3.4.3'
-  id 'org.owasp.dependencycheck' version '12.1.0'
+  id 'org.springframework.boot' version '3.4.4'
+  id 'org.owasp.dependencycheck' version '12.1.1'
```

Comment:
I wonder we should leave the dependency checker at v10.0.3 and add suppressions as needed like PIP. I am not too fussed just need to agree what is best to do :)